x-pack/plugins/apm/public/components/shared/Stacktrace/index.tsx

```diff
@@ -79,7 +79,7 @@ interface StackframesGroup {
 
 export function getGroupedStackframes(stackframes: IStackframe[]) {
   return stackframes.reduce((acc, stackframe) => {
-    const prevGroup = last(acc);
+    const prevGroup = last(acc) as any;
```

Comment:
You can avoid `any` by changing this line:

```diff
- if (shouldAppend) {
+ if (prevGroup && shouldAppend) {
```

Reply:
👍